gvr-modelviewer2/modelviewer2/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'com.android.application'
+
+allprojects {
+    repositories {
+        jcenter()
+        flatDir {
+            dirs '../../gearvrf-libs'
+        }
+    }
+}
+
+android {
+    compileSdkVersion 19
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        //applicationId "com.samsung.android.gvr.sample"
+        applicationId "org.gearvrf.ModelViewer2Activity"
+        minSdkVersion 19
+        targetSdkVersion 19
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+
+        }
+    }
+
+    sourceSets.main {
+        jni.srcDirs = [] // no auto generation of Android.mk
+        jniLibs.srcDir 'libs' // pre-compiled libraries
+    }
+}
+
+dependencies {
+    compile(name: 'framework-debug', ext: 'aar')
+    compile(name: 'widget_plugin-debug', ext: 'aar')
```

Comment:
@apjagdale Please change widget_plugin-debug to widgetplugin-debug so it can build without modifications.